test_mixpanel.py

```diff
@@ -25,16 +22,28 @@ def send(self, endpoint, event, api_key=None):
         else:
             self.log.append((endpoint, json.loads(event)))
 
-class MixpanelTestCase(unittest.TestCase):
-    def setUp(self):
-        self.TOKEN = '12345'
+
+# Convert a query string with base64 data into a dict for safe comparison.
+def qs(s):
```

Comment:
This name may be hard to grok, since it isn't just about a query string (for example, it assumes that it gets a query string always of the form `data=base64....` - is there a more descriptive name that doesn't make you type too much?